--- Pysolar/rest.py.bak	2015-06-17 11:26:42.176959550 -0700
+++ Pysolar/rest.py	2015-06-17 11:27:44.102958765 -0700
@@ -36,7 +36,7 @@
 def GetAerosolOpticalDepth(turbidity_beta, effective_wavelength, turbidity_alpha):
 	# returns tau_a
 	print("effective_wavelength: ")
-	print effective_wavelength
+	print(effective_wavelength)
 	return turbidity_beta * effective_wavelength ** -turbidity_alpha
 
 def GetAerosolScatteringCorrectionFactor(band, ma, tau_a):
