--- ./shapely/geos.py	2015-04-29 21:27:19.000000000 +0100
+++ ./shapely/geos.py	2015-05-10 10:20:18.000000000 +0100
@@ -67,21 +67,7 @@
     if os.path.exists(geos_whl_dylib):
         _lgeos = CDLL(geos_whl_dylib)
     else:
-        if hasattr(sys, 'frozen'):
-            try:
-                # .app file from py2app
-                alt_paths = [os.path.join(os.environ['RESOURCEPATH'],
-                            '..', 'Frameworks', 'libgeos_c.dylib')]
-            except KeyError:
-                # binary from pyinstaller
-                alt_paths = [os.path.join(sys.executable, 'libgeos_c.dylib')]
-        else:
-            alt_paths = [
-                # The Framework build from Kyng Chaos
-                "/Library/Frameworks/GEOS.framework/Versions/Current/GEOS",
-                # macports
-                '/opt/local/lib/libgeos_c.dylib',
-            ]
+        alt_paths = [os.path.join(sys.prefix, 'lib', 'libgeos_c.dylib')]
         _lgeos = load_dll('geos_c', fallbacks=alt_paths)
 
     free = load_dll('c').free
